doc/mgr/prometheus.rst

```diff
@@ -152,6 +153,13 @@ The metrics take the following form:
     ceph_health_detail{name="OSD_DOWN",severity="HEALTH_WARN"} 1.0
     ceph_health_detail{name="PG_DEGRADED",severity="HEALTH_WARN"} 1.0
 
+The module also maintains an in-memory history of health check stats
+By default, it retains a maximum of 1000 entries, These limits are configurable via the following runtime option:
```

Comment:
Something strange going on with the commas / capitalization and I suggest to rewrite the beginning of the first sentence (but I might be wrong). And because now there is only 1 option instead of the 2 before, talk about a singular "limit":

```suggestion
By default the history retains a maximum of 1000 entries. This limit is configurable via the following runtime option:
```